src/test/java/com/google/api/codegen/testdata/java_page_streaming_response_library.baseline

```diff
@@ -47,43 +47,282 @@ import javax.annotation.Generated;
 @ExperimentalApi
 public class PagedResponseWrappers {
 
-  public static class ListShelvesPagedResponse extends PagedListResponseImpl<ListShelvesRequest, ListShelvesResponse, Shelf> {
+  public static class ListShelvesPagedResponse implements PagedListResponse<Shelf> {
+
+    private final PagedListResponseContext<RequestT, ResponseT, ResourceT> context;
+    private final ListShelvesPage page;
 
     public ListShelvesPagedResponse(
         UnaryCallable<ListShelvesRequest, ListShelvesResponse> callable,
         PagedListDescriptor<ListShelvesRequest, ListShelvesResponse, Shelf> pageDescriptor,
         ListShelvesRequest request,
         CallContext context) {
-      super(callable, pageDescriptor, request, context);
+      this.context = new PagedListResponseContext<>(callable, pageDescriptor, request, context);
+      this.page = new ListShelvesPage(context);
+    }
+
+    public Iterator<Shelf> iterateAll() {
+      return context.iterateAll();
+    }
+
+    public Page<Shelf> getPage() {
+      return page;
+    }
+
+    public Iterator<ListShelvesPage> iteratePages() {
+      return new PageContext.PageIterator<ListShelvesPage>(
+        new PageContext.PageFetcher<>() {
+          @Override
+          public ListShelvesPage getNextPage(ListShelvesPage currentPage) {
+            return currentPage.getNextPage();
+          }
+      }
+    }
+
+    public Object getNextPageToken() {
```

Comment:
I think there was going to be some kind of standardization on the page token type, instead of allowing for both String and ByteString. Let's figure out what it is. 

Related: https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1764